app/agent/origin.scala

```diff
@@ -136,4 +138,6 @@ case class JsonOrigin(vendor:String, account:String, url:String, resources:Set[S
     Json.parse(jsonText)
   }
   val jsonFields = Map("url" -> url)
+
+  override def toMarkerMap: Map[String, Any] = jsonFields
```

Comment:
Where you see `toMarkerMap` set to `jsonFields`, we're doing that just to compile. We don't actually use these values!